components/d2l-applied-filters/d2l-applied-filters.js

```diff
@@ -127,7 +126,7 @@ class D2lAppliedFilters extends RtlMixin(LocalizeStaticMixin(LitElement)) {
 						text="${x.text}"
 						deletable
 						index="${index}"
-						@d2l-labs-multi-select-list-item-deleted="${() => this._multiSelectItemDeleted(x)}"
+						@d2l-labs-multi-select-list-item-deleted="${this._multiSelectItemDeleted}"
```

Comment:
This failed the [`no-template-arrow`](https://github.com/43081j/eslint-plugin-lit/blob/HEAD/docs/rules/no-template-arrow.md) rule.  This results in the small rewrite in `_multiSelectItemDeleted` below